src/charm.py

```diff
@@ -25,6 +26,12 @@ def __init__(self, *args: typing.Any) -> None:
         """
         super().__init__(*args)
         self.framework.observe(self.on.config_changed, self.config_service)
+        self.ingress = IngressPerAppRequirer(
+            self,
+            port=8000,
```

Comment:
I think we should define a constant for the port, it's repeated

Reply:
Fixed